anoncreds/src/ffi/credential.rs

```diff
@@ -60,6 +60,7 @@ pub extern "C" fn anoncreds_create_credential(
     attr_raw_values: FfiStrList,
     attr_enc_values: FfiStrList,
     rev_reg_id: FfiStr,
+    rev_reg_list: ObjectHandle,
```

Comment:
This is called `rev_reg_list` and in another method there's `rev_status_list`. Are those both referring to the same?

Reply:
As long as it's consistent. But yes, `rev_status_list` sounds good